AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ReconcileAndLocalSaveQueue.swift

```diff
@@ -71,13 +71,15 @@ class ReconcileAndSaveQueue: ReconcileAndSaveOperationQueue {
     }
 
     func addOperation(_ operation: ReconcileAndLocalSaveOperation, modelName: String) {
+
         serialQueue.async {
             var reconcileAndLocalSaveOperationSink: AnyCancellable?
-
             reconcileAndLocalSaveOperationSink = operation.publisher.sink { _ in
```

Comment:
This code requires Swift 5.3 in the the latest version of XCode 12.  To restore compatibility with XCode 11.4, which is the minimum version indicated in the README, you can change it to:

`
            reconcileAndLocalSaveOperationSink = operation.publisher.sink( receiveCompletion: { _ in
                self.serialQueue.async {
                    self.reconcileAndLocalSaveOperationSinks.with { $0.remove(reconcileAndLocalSaveOperationSink) }
                    self.modelReconcileAndSaveOperations[modelName]?[operation.id] = nil
                    self.reconcileAndSaveQueueSubject.send(.operationRemoved(id: operation.id))
                }
            }, receiveValue: { _ in } )
`